shared/src/main/java/uk/gov/di/authentication/shared/services/AccessTokenService.java

```diff
@@ -81,4 +83,19 @@ public Optional<AccessTokenStore> setAccessTokenTtlTestOnly(String accessToken,
                             return ts;
                         });
     }
+
+    @Override
+    public Optional<AccessTokenStore> get(String partition) {
+        Key partitionKey = Key.builder().partitionValue(partition).build();
+        Optional<AccessTokenStore> accessTokenStore =
+                Optional.ofNullable(dynamoTable.getItem(partitionKey));
+
+        if (accessTokenStore.isPresent()) {
+            return accessTokenStore;
+        } else {
+            GetItemEnhancedRequest getItemEnhancedRequest =
+                    GetItemEnhancedRequest.builder().key(partitionKey).consistentRead(true).build();
+            return Optional.ofNullable(dynamoTable.getItem(getItemEnhancedRequest));
```

Comment:
I don't think there's any way from this to work out how often we're retrying, and how often that is making a difference